skaffold.yaml

```diff
@@ -9,40 +20,95 @@ build:
   artifacts:
   - image: valory/open-autonomy
     context: deployments/Dockerfiles/autonomy
-    docker:
+    docker: 
       dockerfile: Dockerfile
-      buildArgs:
-        AEA_VERSION: "1.17.0"
-
-
+      buildArgs: 
+        <<: *aea_version
 profiles:
   - name: release
     build:
-        tagPolicy:
-          envTemplate:
-            template: "{{.VERSION}}"
-        artifacts:
-
-        - image: valory/open-autonomy
-          context: deployments/Dockerfiles/autonomy
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-            buildArgs:
-              AEA_VERSION: "1.17.0"
-
-        - image: valory/open-autonomy-tendermint
-          context: deployments/Dockerfiles/tendermint/
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-
-        - image: valory/open-autonomy-hardhat
-          context: deployments/Dockerfiles/hardhat
-          docker:
-            noCache: true
-
-        - image: valory/open-autonomy-docs
-          docker:
-            noCache: true
-            dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+      tagPolicy:
+        envTemplate:
+          template: "{{.VERSION}}"
```

Comment:
I think this is actually unnecessary now as well.

Reply:
Why?

Reply:
Because this is `release-latest` profile, so we don't need to pick up the version from env anymore. I've fixed this here https://github.com/valory-xyz/open-autonomy/pull/1274/commits/e4333413badb4d90f5ce8986f157bdaf5897ba93